examples/effects_clipping_caps.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
```

Comment:
@shrekshao can you split this into two examples: (1) with intersecting clip planes that shows the artifact, and (2) with non-intersecting clip planes that has no artifact. We would tweet/feature (2), and in the info text for (2) there would be a caveat with a link to (1).